gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/task/HiveConverterUtils.java

```diff
@@ -151,6 +154,30 @@ public static String generateCreateDuplicateTableDDL(
         dbName, tblName, inputDbName, inputTblName, tblLocation);
   }
 
+  public static String generateAlterSchemaDML(
+      String tableName,
+      Optional<String> optionalDbName,
```

Comment:
Since I try to make it consistent with method generateCreateDuplicateTableDDL. In which the dbName is optional. I guess it's for testing. But not so sure about that

Reply:
+1 for consistency.